shared/ui/src/commonMain/kotlin/com/adammcneilly/pocketleague/shared/ui/feed/FeedEventGroup.kt

```diff
@@ -0,0 +1,37 @@
+package com.adammcneilly.pocketleague.shared.ui.feed
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import com.adammcneilly.pocketleague.core.displaymodels.EventGroupDisplayModel
+import com.adammcneilly.pocketleague.shared.ui.event.EventSummaryListCard
+import com.adammcneilly.pocketleague.shared.ui.event.LanEventSummaryCard
+import com.adammcneilly.pocketleague.shared.ui.utils.screenHorizontalPadding
+
+/**
+ * For a given [displayModel], determine how to render
+ * that collection for our [FeedContent].
+ */
+@Composable
+fun FeedEventGroup(
```

Comment:
Maybe worth adding a paparazzi test for this? Seems like it'd be a little better for a UI test to verify the conditional logic instead of paparazzi.